maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/controller/PassportAssessmentController.java

```diff
@@ -0,0 +1,94 @@
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/passport-assessments")
+@Slf4j
+@RequiredArgsConstructor
+@Tag(name = "Passport Assessments", description = "Rest API for passport assessments")
+public class PassportAssessmentController {
+
+    private Gson gson;
+    private final GsonBuilder gsonBuilder;
+    private final PassportAssessmentService passportAssessmentService;
+    private final PassportAssessmentValidationProcessor passportAssessmentValidationProcessor;
+
+    @PostConstruct
+    private void initialize() {
+        gson = gsonBuilder
+                .setPrettyPrinting()
+                .serializeNulls()
+                .registerTypeAdapter(LocalDateTime.class, new DateUtil.LocalDateTimeAdapter())
+                .create();
+    }
+
+    @GetMapping(value = "/{passportAssessmentId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve a passport assessment record")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE)),
+            @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class))),
+            @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
+    })
+    public ResponseEntity<Object> getAssessment(@PathVariable int passportAssessmentId) {
+        log.info("Get Passport Assessment Request Received");
+        passportAssessmentValidationProcessor.validate(passportAssessmentId);
+        PassportAssessmentDTO passportAssessment = passportAssessmentService.find(passportAssessmentId);
+        String passportAssessmentJson = gson.toJson(passportAssessment, PassportAssessmentDTO.class);
+        return ResponseEntity.ok(passportAssessmentJson);
+    }
+
+    @PutMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Update a passport assessment record")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE)),
+            @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class))),
+            @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorDTO.class)))
+    })
+    public ResponseEntity<Object> updateAssessment(
+            @Parameter(description = "Passport assessment data", content = @Content(mediaType = "application/json",
```

Comment:
There is a mix of MediaType.APPLICATION_JSON_VALUE and application/json, it might be better if we stuck with the prior when specifying all media types.